packages/measure-tools/README.md

```diff
@@ -0,0 +1,164 @@
+# @bentley/measure-tools-react
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+The measure-tools-react package provides measurement tools and React components to access within an iTwin Viewer.
+
+Every iTwin.js application has initialization code where they must call ```IModelApp.startup()```. This library has a similar initialization entrypoint that must be called to configure localization and register tools.
+Call the  ```MeasureTools.startup() ``` **AFTER** the IModelApp one.
+
+```typescript
+    await IModelApp.startup(opts);
+    await MeasureTools.startup(opts);
+```
+
+After that, most of your code changes will be to consume the default measurement tools (such as measure distance, polygon, location, etc).
+
+**Note:** Startup can be called any number of times but will only startup once. Some other libraries that use measure-tools may have already initialized it for you!
+
+## What to add in your iTwin-Viewer based app (iTwin App UI)
+
+With a few short lines, you can add the measurement tools and widgets to your app. The iTwin Viewer accepts a callback, `onIModelAppInit`, which you can use to initialize the MeasureTools. Afterwards, all you need to do is pass in the `MeasureToolsUiItemsProvider` to the list of UiProviders for your viewer to register. This will add both the toolbar with all the measure tools and the measurement property widget to your app for you.
+
+```typescript
+  const handleOnIModelAppInit = async () => {
+    await MeasureTools.startup();
+  };
+
+  return (
+    <Viewer
+      ...
+      onIModelAppInit={handleOnIModelAppInit}
+      uiProviders={[
+        new MeasureToolsUiItemsProvider(),
+      ]}
+    />
+  );
+```
+
+## What to add in your frontstage
+
+If you are not using the iTwin-Viewer or iTwin App UI, you can add the measurement tools and measurement property widget to your frontstage like so:
```

Comment:
Is this going to describe the UI 1.0 way of doing it, or is it agnostic to the UI layer?

Reply:
This is the way to add to your ui-1.0 based app. Technically it will work in ui-2.0 as well, but we don't encourage this, devs should be using UiItemProviders to add to their frontstage instead of directly like we do here in 1.0. Also i dont believe we technically expose the term "ui-1.0" externally so i didn't want to add to more confusion. I guess we could technically remove this chunk altogether.... Thoughts?

Reply:
> I guess we could technically remove this chunk altogether.... Thoughts?

That's what I was debating when originally writing the comment. I'm in favor of doing that since it's pretty much how everything should be setup going forward if you're using our React UI. If you're not using our React UI pkgs then this doesn't help you much.